internal/factsengine/gatherers/sbddump.go

```diff
@@ -0,0 +1,50 @@
+package gatherers
+
+import (
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/utils"
+)
+
+const (
+	SBDDumpFactKey = "SBD_dump"
+)
+
+type SBDDumpGatherer struct {
+	executor CommandExecutor
+}
+
+func NewSBDDumpGatherer() *SBDDumpGatherer {
+	return &SBDDumpGatherer{
+		executor: Executor{},
+	}
+}
+
+func (s *SBDDumpGatherer) Gather(factsRequests []FactRequest) ([]Fact, error) {
+	facts := []Fact{}
+	log.Infof("Starting %s facts gathering process", SBDDumpFactKey)
+
+	for _, factReq := range factsRequests {
+		args := strings.Split(factReq.Argument, ":")
+		SBDDump, err := s.executor.Exec(
+			"sbd", "dump", "-d", args[0], "dump", "-b")
```

Comment:
I don't understand this command. The command should be `sbd -d /dev/sdc dump`. What is this `dump -b` at the end.